### DISCUSSION TITLE
osu lazer doesn't pop up

### DISCUSSION THREAD

**Discussion by paletkademon** [ppy/osu#17882] [Q&A]
hi i downloaded osu lazer and it doesn't pop up, the installation was succesful but i can't open the game, it pop up on task manager but for only seconds. i don;t know what to do?

**Comment by ZenFlora** [comment]
I have the same problem, what windows are you on?

**Reply by paletkademon** [reply]
windows 10 home

**Reply by frenzibyte** [reply]
@ProtogenAxion your issue may be 100% different depending on the logs, so please attach as well (#15188)

**Comment by frenzibyte** [comment]
https://github.com/ppy/osu/discussions/15188

**Reply by paletkademon** [reply]
?

**Reply by paletkademon** [reply]
[performance.log](https://github.com/ppy/osu/files/8515009/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8515010/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8515011/updater.log)
[database.log](https://github.com/ppy/osu/files/8515012/database.log)
[network.log](https://github.com/ppy/osu/files/8515013/network.log)
this

**Reply by frenzibyte** [reply]
Nothing shown in logs, please check if you can find anything relevant in the Event Viewer.

**Comment by paletkademon** [comment]
?


wt., 19 kwi 2022 o 19:15 Salman Ahmed ***@***.***> napisał(a):

> Nothing shown in logs, please check if you can find anything from the
> Windows Event Viewer.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ppy/osu/discussions/17882#discussioncomment-2595027>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AYZOWEXLGHQBV5JHJRFU3O3VF3SZXANCNFSM5TYMS7NQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Pozdrawiam Aleks Armatys

**Comment by bdach** [comment]
Please attach Windows Event Viewer info by performing the following steps:
1. Type `Event viewer` in the Windows search bar (shortcut: <kbd>Windows</kbd>+<kbd>S</kbd>) and press <kbd>Enter</kbd>, or open the *Run...* button (<kbd>Windows</kbd>+<kbd>R</kbd>), type in `eventvwr` and press <kbd>Enter</kbd>.
2. The *Event viewer* window will appear. Expand *Windows Logs* on the left and choose *Application*. It might take a while to load.
3. In the *Actions* pane on the right click *Save All Events As...*, then choose a location to save to.
4. On the *Display Information* popup, select the second option, then the checkbox to show all available languages, then *English (UK)* (US would also do fine).
5. Append the resulting `.evtx` file to this issue.
